src/main/java/frc/robot/Constants.java

```diff
@@ -276,6 +290,64 @@ public static class constShooter {
     public static final Measure<Angle> ZEROED_ANGLE = Units.Degrees.of(0);
 
     public static final Measure<Time> ZEROING_TIMEOUT = Units.Seconds.of(3);
+
+    public static class ShooterPositionGroup {
+      public Measure<Angle> shooterAngle;
+      public Measure<Velocity<Angle>> leftVelocity, rightVelocity;
+
+      public ShooterPositionGroup(Measure<Angle> shooterAngle, Measure<Velocity<Angle>> leftVelocity,
+          Measure<Velocity<Angle>> rightVelocity) {
+        this.shooterAngle = shooterAngle;
+        this.leftVelocity = leftVelocity;
+        this.rightVelocity = rightVelocity;
+      }
+    }
+
+    public static final ShooterPositionGroup PREP_NONE = new ShooterPositionGroup(
+        constShooter.PIVOT_BACKWARD_INTAKE_LIMIT,
+        Units.RotationsPerSecond.zero(), Units.RotationsPerSecond.zero());
+    public static final ShooterPositionGroup PREP_AMP_SHOOTER = new ShooterPositionGroup(constShooter.PIVOT_AMP_ANGLE,
+        constShooter.LEFT_AMP_VELOCITY, constShooter.RIGHT_AMP_VELOCITY);
+    public static final ShooterPositionGroup PREP_SHUFFLE = new ShooterPositionGroup(constShooter.PIVOT_SHUFFLE_ANGLE,
+        constShooter.LEFT_SHUFFLE_VELOCITY, constShooter.RIGHT_SHUFFLE_VELOCITY);
+    public static final ShooterPositionGroup PREP_SUB = new ShooterPositionGroup(constShooter.PIVOT_SUB_ANGLE,
+        constShooter.LEFT_SUB_VELOCITY, constShooter.RIGHT_SUB_VELOCITY);
+    public static final ShooterPositionGroup PREP_SPIKE = new ShooterPositionGroup(constShooter.PIVOT_SPIKE_ANGLE,
+        constShooter.LEFT_SPIKE_VELOCITY, constShooter.RIGHT_SPIKE_VELOCITY);
+    public static final ShooterPositionGroup PREP_WING = new ShooterPositionGroup(constShooter.PIVOT_WING_ANGLE,
+        constShooter.LEFT_WING_VELOCITY, constShooter.RIGHT_WING_VELOCITY);
+
+  }
+
```

Comment:
Is there a need to create constants only to pass them to another constant? Why not just store only in this object?